src/components/TaskList.js

```diff
@@ -25,12 +25,13 @@ const TaskList = (props) => {
         setTasks(taskArray);
     }
 
-    // Fetch tasks on component mount
+    //updates tasklists when task is added
     useEffect(() => {
+        console.log(`fetching tasks in useEffect: ${roomID}`);
         fetchTasks();
-    // eslint-disable-next-line
-    }, [arrayOfActiveTasks]);
-    
+        // eslint-disable-next-line
```

Comment:
Same question as above, what's the problem here that we are disabling eslint for?

Reply:
This one asks for fetchTasks to be in the dependency, but that causes the hook to loop indefinitely.